ndbench-core/src/test/java/org/libex/test/TestBase.java

```diff
@@ -0,0 +1,56 @@
+package org.libex.test;
+
+import org.hamcrest.Matcher;
+import org.junit.Rule;
+import org.junit.rules.ExpectedException;
+import org.libex.test.google.NullPointerTester;
+import org.libex.test.rules.CheckableErrorCollector;
+
+import javax.annotation.Nullable;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.CoreMatchers.sameInstance;
+
+/**
+ * JUnit test base class.
+ * 
+ * @author John Butler

```

Comment:
that's not me. that is the original author of the library for junit rule for testing logger activity.
i spoke w/ Vinay about using that library.. It works really well, but there is no binary artifact corresponding to the library in maven repo land yet.  Vinay said to commit the source for now.    But if you guys don't think that's the right approach I can look for another way to do the unit test i pushed wi/ this commit.    pls let me know.